src/urllib3/util/request.py

```diff
@@ -5,6 +5,13 @@
 from ..exceptions import UnrewindableBodyError
 
 ACCEPT_ENCODING = 'gzip,deflate'
+try:
+    import brotli
+except ImportError:
+    brotli = None
+if brotli is not None:
```

Comment:
We can roll this `+=` logic within the `try`/`except`.

Reply:
Sure